pkgs/applications/office/ledger-autosync/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, lib, python3Packages, fetchurl }:
+
+python3Packages.buildPythonApplication rec {
+  version = "1.0.0";
+  src = fetchurl {
+    url = "https://github.com/egh/ledger-autosync/archive/v${version}.tar.gz";
+    sha256 = "603d768c2b56c0238ee0abc9fb54ffb7cfac7573ffc484db3d94e211e9cf5453";
+  };
+  name = "ledger-autosync";
```

```suggestion
  pname = "ledger-autosync";
```
`pname` and `version` get combined into `name`.

I'd also prefer if you ordered them like
```
pname = "...";
version = "...";

src =
...
```

Also consistent whitespacing and ordering as you did in `fuzzywuzzy`.